config/cesm/config_grids.xml

```diff
@@ -1920,9 +1862,112 @@
   </domains>
 
   <!-- ======================================================== -->
-  <!-- Mapping -->
+  <!-- ATM/LND domains for single point mct driver -->
   <!-- ======================================================== -->
 
+  <domains driver="mct">
+    <domain name="01col">
+      <nx>1</nx> <ny>1</ny>
+      <file>domain.ocn.01col.ArcticOcean.20150824.nc</file>
+      <file>domain.ocn.01col.ArcticOcean.20150824.nc</file>
+      <desc>01col is a single-column grid for datm and POP:</desc>
+    </domain>
+    <domain name="CLM_USRDAT">
```

Comment:
Because of #3576 I recommend you remove this setting for CLM_USRDAT for a specific domain file.

Reply:
Done

Reply:
Perfect. Thank you!